api/app/Console/Commands/SubmittedAtApplicationDates.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\PoolCandidate;
+use Illuminate\Console\Command;
+use Carbon\Carbon;
+use Database\Helpers\ApiEnums;
+
+class SubmittedAtApplicationDates extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'update-submitted-at';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'For pool candidates that should have submitted dates, submit them';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $candidatesToSubmit = PoolCandidate::where('pool_candidate_status', ApiEnums::CANDIDATE_STATUS_NEW_APPLICATION)
+            ->where('submitted_at', null)->get();
+
+        foreach ($candidatesToSubmit as $candidate) {
```

Comment:
You can do a mass update without a for loop - probably a lot faster! https://laravel.com/docs/10.x/eloquent#mass-updates

Reply:
implemented! https://github.com/GCTC-NTGC/gc-digital-talent/pull/6041/commits/2b56645e966205e52093dc632e6e23c51ef514cd